pystiche/papers/sanakoyeu_et_al_2018/utils.py

```diff
@@ -0,0 +1,72 @@
+from typing import Any, List, Optional, Union
+
+import torch
+from torch import nn, optim
+from torch.optim.lr_scheduler import ExponentialLR
+from torch.optim.optimizer import Optimizer
+
+from pystiche.ops.container import OperatorContainer
+from pystiche.ops.op import ComparisonOperator
+from pystiche.optim.meter import FloatMeter
+
+
+def sanakoyeu_et_al_2018_optimizer(
+    params: Union[nn.Module, List[torch.Tensor]]
+) -> Optimizer:
+    if isinstance(params, nn.Module):
+        params = params.parameters()
+    return optim.Adam(params, lr=2e-4)
+
+
+def sanakoyeu_et_al_2018_lr_scheduler(optimizer: Optimizer) -> Optional[ExponentialLR]:
+    return DelayedExponentialLR(optimizer, gamma=0.1, delay=2)
+
+
+class ExponentialMovingAverage(FloatMeter):
+    def __init__(self, name: str, init_val: float = 0.8, alpha: float = 0.05):
+        super().__init__(name)
+        self.alpha = alpha
```

Comment:
We have discussed this before: avoid symbol names as variable names. Please check if there is some agreed upon name for this. If that is not the case I suggest something along the lines of `discount_factor`. No strong opinion here, though.